### DISCUSSION TITLE
Page doesn't render components after loading data from data base

### DISCUSSION THREAD

**Discussion by CH4R4F** [GrapesJS/grapesjs#4397] [Q&A] [answered]
I'm storing pages data in a database (MYSQL) using php
in the urlLoad I load data correctly as a json string .. but instead of rendering the page, it shows me the response in the canvas
![Screenshot 2022-06-19 at 14-15-57 Buildfy test](https://user-images.githubusercontent.com/74823355/174483078-c96fea95-0198-46fb-95c0-38bbee217bd0.png)

this is the code
![Screenshot 2022-06-19 142101](https://user-images.githubusercontent.com/74823355/174483151-297b8851-3fdd-428b-aba2-a655cea9478c.png)

**Comment by artf** [comment, marked as answer]
Hi @CH4R4F if you're using the latest version please check https://grapesjs.com/docs/modules/Storage.html as a few configurations have been changed.

**Reply by clonefunnels** [reply]
This is SOOOO confusing.  I'm a pretty good programmer and I can't figure it out.
